config/initializers/core_extensions/devise/strategies/json_web_token.rb

```diff
@@ -5,7 +5,7 @@ module Strategies
     # strategy for login with jwt
     class JsonWebToken < Base
       def valid?
-        return false unless request.headers['Authorization'].present?
+        return false unless request.headers['HTTP_AUTHORIZATION'].present?
```

Comment:
Not sure if this change is necessary (probably not) or what the rails convention is for using `Authorization` vs `HTTP_AUTHORIZATION`.